numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1558,6 +1570,35 @@ pack_inner(const char *inptr,
             inptr += 16;
         }
     }
+#elif defined __ARM_NEON || defined __ARM_NEON__
+    if (in_stride == 1 && element_size == 1 && n_out > 2) {
+        /* don't handle non-full 8-byte remainder */
+        npy_intp vn_out = n_out - (remain ? 1 : 0);
+        vn_out -= (vn_out & 1);
+        for (index = 0; index < vn_out; index += 2) {
+            unsigned int r;
+            npy_uint64 a = *(npy_uint64*)inptr;
+            npy_uint64 b = *(npy_uint64*)(inptr + 8);
+            if (order == 'b') {
+                a = npy_bswap8(a);
+                b = npy_bswap8(b);
+            }
+            const npy_uint64 __attribute__((aligned(16))) data[2] = {a, b};
+            uint64x2_t v = vld1q_u64(data);
```

Comment:
use vcreate_u64 and vcombine_u64 instead.

Reply:
modified.